templates/desktop/organisations.html

```diff
@@ -560,194 +565,44 @@ <h3>Landscape</h3>
     </div>
     <div class="col-6">
       <h3>Active Directory</h3>
-      <p>Integrating Ubuntu Desktop into an existing Active Directory architecture can be an automated and effortless process when using System Security Services Daemon (SSSD). GPO is available by default to manage desktop settings and Ubuntu Advantage users receive access to additional Active Directory policies like remote script execution and privilege management.</p>
-      <p><a href="/engage/microsoft-active-directory">Integrating Ubuntu Desktop with Active Directory&nbsp;&rsaquo;</a></p>
-      <p><a href="https://github.com/ubuntu/adsys/wiki">ADSys documentation&nbsp;&rsaquo;</a></p>
+      <p>Integrating Ubuntu Desktop into an existing Active Directory architecture can be an automated and effortless process when using System Security Services Daemon (SSSD). GPO is available by default to manage desktop settings and Ubuntu Pro users receive access to additional Active Directory policies like remote script execution and privilege management.</p>
+      <p><a href="/engage/microsoft-active-directory" class="p-button--positive">Integrating Ubuntu Desktop with Active Directory</a></p>
+      <p><a href="https://github.com/ubuntu/adsys/wiki" class="p-button">ADSys documentation</a></p>
     </div>
   </div>  
 </section>
-
-<section class="p-strip">
-  <div class="u-fixed-width">
-    <h2>Enterprise-grade support with Ubuntu Advantage</h2>
-    <p>Ubuntu Advantage Desktop is a single, per-seat package delivering comprehensive enterprise security, management tooling, and extended support for your Ubuntu Desktop fleet.</p>
-  </div>
-  <div class="row">
-    <div class="col-4 u-sv3">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/ceb481d1-10+years.svg",
-        alt="",
-        width="100",
-        height="100",
-        hi_def=True,
-        loading="lazy",
-        attrs={"class": "u-hide--small u-hide--medium"}
-        ) | safe
-      }}
-      <h3>Enterprise-grade security</h3>
-      <p>Receive 10 years of security patches and updates for Ubuntu LTS releases with Extended Security Maintenance (<a href="/security/esm">ESM</a>). Minimise downtime with Kernel <a href="/security/livepatch">Livepatch</a>.</p>
+<section class="p-strip is-bordered">
```

Comment:
I've build this same section twice on other pages, it might be worth breaking it out into an include module. If you do, use your version as it looks better than the ones I made.

Reply:
I can't find the same sections at the moment maybe because you are still working on it and it hasn't been merged it to main yet? Could we break it into module later?